system/modules/news/modules/ModuleNewsList.php

```diff
@@ -162,4 +162,60 @@ protected function compile()
 
 		$this->Template->archives = $this->news_archives;
 	}
+
+	/**
+	 * Counts the total matching items
+	 *
+	 * @param   array $newsArchives
+	 * @param   boolean $blnFeatured
+	 *
+	 * @return  int
+	 */
+	protected function countItems($newsArchives, $blnFeatured)
+	{
+		// HOOK: add custom logic
+		if (isset($GLOBALS['TL_HOOKS']['newsListCountItems']) && is_array($GLOBALS['TL_HOOKS']['newsListCountItems']))
+		{
+			foreach ($GLOBALS['TL_HOOKS']['newsListCountItems'] as $callback)
+			{
+				$intResult = \System::importStatic($callback[0])->$callback[1]($newsArchives, $blnFeatured, $this);
+
+				if ($intResult !== false && is_int($intResult))
```

Comment:
Why are you checking for `!== false` if `$intResult` has to go through `is_int()`? Does `is_int()` return `true` if you pass `false` as argument?